Examples/Examples/WorldScaleExampleView.swift

```diff
@@ -63,6 +63,7 @@ struct WorldScaleExampleView: View {
         .onSingleTapGesture { _, scenePoint in
             graphicsOverlay.addGraphic(Graphic(geometry: scenePoint))
         }
+        .ignoresSafeArea(edges: .bottom)
```

Comment:
```suggestion
        .ignoresSafeArea(edges: [.horizontal, .bottom])
```
![IMG_3580](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/9660181/36e22786-d35c-4d6f-8e9e-0611d5666491)

In the rare case when the app is used in landscape mode, we still need horizontal safe area ignore. Otherwise it won't extend into the shorter edges of the screen.